src/users/users.service.ts

```diff
@@ -10,9 +10,20 @@ export class UsersService {
 
   constructor( @InjectModel(User.name) private userModel: Model<UserDocument> ) {}
 
-  async create(createUserDto: CreateUserDto): Promise<User> {
+  async create(createUserDto: CreateUserDto): Promise<User> {  
 
-    return new this.userModel(createUserDto).save();
+            return  new this.userModel(createUserDto).save();
+  
+}
+
+  async userLogin(createUserDto: User): Promise<User>  {
+    console.log(createUserDto);
```

Comment:
Once testing in done with console. remove the console or if it required at staging add a comment